internal/storage/postgreSQL/storage.go

```diff
@@ -0,0 +1,45 @@
+package postgreSQL
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/green-ecolution/green-ecolution-backend/config"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage/postgreSQL/Schema"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage/postgreSQL/sensorSQL"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage/postgreSQL/treeSQL"
+	_ "github.com/lib/pq"
+	"log"
+)
+
+func NewPostgresDB(ctx context.Context, cfg config.DatabaseConfig) (*storage.Repository, error) {
```

Comment:
Please change the name to `NewRepository`

```suggestion
func NewRepository(ctx context.Context, cfg config.DatabaseConfig) (*storage.Repository, error) {
```